stubs/requests/requests/api.pyi

```diff
@@ -1,142 +1,150 @@
 from typing import Any
 
 from .models import Response
-from .sessions import _Data, _Params
+from .sessions import RequestsCookieJar, _Auth, _Cert, _Data, _Files, _HooksInput, _Params, _TextMapping, _Timeout, _Verify
 
 def request(
-    method: str | bytes,
+    method: str,
```

Comment:
Why change this? Seems like `bytes` works fine, even if it is a little weird:

```python
>>> import requests
>>> requests.request(b'GET', 'https://www.bbc.com/news')
<Response [200]>
```

Reply:
Ah, you're right! Changed it back.

The implementation for `url` has an explicit check+conversion for `bytes`, which is not there for `method`. They also call `.upper()` on it, but apparently that also works for `bytes` :-)